tests/Context/ServerOsContextTest.php

```diff
@@ -83,39 +83,39 @@ public function valuesDataProvider(): array
                 ],
                 [],
                 UndefinedOptionsException::class,
-                'The option "foo" does not exist. Defined options are: "build", "kernel_version", "name", "version".',
+                '/The option "foo" does not exist. Defined options are: "build", "kernel_version", "name", "version"./',
```

Comment:
As this is a regex now, you should escape all special characters like the dot. Also, please use `^` and `$` to make sure that the matching is done against the string as a whole